caikit/runtime/http_server/http_server.py

```diff
@@ -760,6 +768,33 @@ def _get_response_openapi(
         output = {200: {"content": response_schema}}
         return output
 
+    def _model_info(
+        self, model_ids: Annotated[List[str], Query(default_factory=list)]
+    ) -> Dict[str, Any]:
+        """Create wrapper for get_models_info so model_ids can be marked as a query parameter"""
```

Comment:
Does this mean it will _always_ be a query parameter or that it _can_ be in a query parameter?

Reply:
I'd still like to understand the semantics of this one

Reply:
@gabe-l-hart sorry I missed this one! It will always be an optional query parameter. From [my understanding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET) it's an anti-pattern to send data in the body of a GET request

Reply:
Ah, yes, that's definitely true. I'm so used to `gRPC` where everything is a POST!